### PR TITLE
Feature/515 new camera parameters

### PR DESCRIPTION
**Pull request by AfrahGh**
* **Tickets addressed:** bsk-515
* **Review:** By commit  
* **Merge strategy:** Merge (no squash)  

## Description
A new camera model message was added. The difference to the old one is a new private field, some of the parameters are a duplicate from the old public field, some have different variable types,  and new ones were also added. The simulation was modified to create and populate the new message.

## Verification
Unit test was updated to test the reading and writing of the camera message 

## Documentation
Documentation was updated with the new camera model message 

## Future work
No future work

### CODE REVIEW

File: src/simulation/sensors/camera/camera.cpp
Comment (patkenneally) [inline src/simulation/sensors/camera/camera.cpp]: :small_red_triangle_down: This should return a `bool`

File: src/simulation/sensors/camera/camera.h
Comment (patkenneally) [inline src/simulation/sensors/camera/camera.h]: :small_red_triangle_down: This should probably be a `bool`.

File: src/simulation/sensors/camera/camera.h
Comment (patkenneally) [inline src/simulation/sensors/camera/camera.h:118]: :large_orange_diamond: This change belongs squashed into the subsequent commit

File: src/architecture/msgPayloadDefCpp/CameraModelMsgPayload.h
Comment (patkenneally) [inline src/architecture/msgPayloadDefCpp/CameraModelMsgPayload.h:4]: :small_red_triangle_down: This copyright statement is incorrect.

File: src/simulation/sensors/camera/camera.cpp
Comment (patkenneally) [inline src/simulation/sensors/camera/camera.cpp:341]: :large_orange_diamond: I'm possibly missing the point (and will dig further), but here we set `imageCadence` to `renderRate` in the message, yet the module also has a `renderRate` variable. Is this because we're placing these functionalities alongside each before we come back and significantly refactor this module in general?

**Review by patkenneally — CHANGES_REQUESTED** [review]
Legend:
:small_red_triangle_down: *Issues* to address before merge
:large_orange_diamond: *Requests* that should not block merge, but should at least be discussed
:large_blue_circle: *Recommendations* that can be ignored if desired

File: src/simulation/sensors/camera/camera.cpp
Comment (thibaudteil) [reply src/simulation/sensors/camera/camera.cpp:341]: That's correct, almost all of the camera parameters are duplicated in order to keep the previous functionality. The camera refactor ticket is alive and well!

File: src/simulation/sensors/camera/camera.h
Comment (thibaudteil) [reply src/simulation/sensors/camera/camera.h:118]: done

File: src/simulation/sensors/camera/camera.cpp
Comment (thibaudteil) [reply src/simulation/sensors/camera/camera.cpp]: done

File: src/simulation/sensors/camera/camera.h
Comment (thibaudteil) [reply src/simulation/sensors/camera/camera.h]: done

File: src/architecture/msgPayloadDefCpp/CameraModelMsgPayload.h
Comment (thibaudteil) [reply src/architecture/msgPayloadDefCpp/CameraModelMsgPayload.h:4]: done

**Pull request merged**
